opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -62,6 +67,62 @@ def collect(self) -> Optional[_PointVarT]:
         pass
 
 
+class _AggregationFactory(ABC):
+    @abstractmethod
+    def _create_aggregation(self, instrument: Instrument) -> _Aggregation:
+        """Creates an aggregation"""
+
+
+class DefaultAggregation(_AggregationFactory):
+    """
+    The default aggregation to be used in a `View`.
+
+    This aggregation will create an actual aggregation depending on the
+    instrument type, as specified next:
+
+    `Counter`                   →   `SumAggregation`
```

Comment:
nit maybe an RST table would look better in the docs

Reply:
Fixed :v: